package.nls.json

```
@@ -101,6 +101,8 @@
     "AWS.command.aboutToolkit": "About Toolkit",
     "AWS.command.downloadLambda": "Download...",
     "AWS.command.uploadLambda": "Upload Lambda...",
+    "AWS.command.uploadToAws": "Upload to AWS...",
+    "AWS.command.uploadToAws.cn": "Upload to AWS...",
```

Like the `.cn` items, "Amazon" must be used instead of "AWS".

```suggestion
    "AWS.command.uploadToAws.cn": "Upload to Amazon...",
```